data/tests/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
+set -e
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd $SCRIPT_DIR
+
+rm -f *.deb
+
+# compression_methods=("xz" "gzip" "none")
+compression_methods=("none")
```

Comment:
For now I only use `none`, I should not commit the other files in the MR.

Reply:
The goal is to parse a real `.deb` file, but not sure it's worth it.
Should I could mock `entry.header().identifier()` or remove the test ?